decentraland/unity-explorer
# chore: add teleport error handling

## What does this PR change?

Loading exceptions were unhandled. Therefore, you could be stuck in the loading screen if something went wrong.

For example, in my tests, for some reason sometimes I cannot reach realms (probably a server hiccup). And I had no way out other than just waiting for the timeout.

This PR adds handling for that case and other possible cases.


<img width="1728" alt="Screen Shot 2024-08-23 at 11 18 22" src="https://github.com/user-attachments/assets/81324d1a-5aaf-42a3-aafd-18b93bdc07c9">


## How to test the changes?


1. Launch the explorer
2. "Break" your internet connection (turn off wifi)
3. Try to go to any realm. The loading screen should not appear
4. "Fix" your internet connection. You should now be able to teleport

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
